encodings/byte_bool/src/stats.rs

```diff
@@ -0,0 +1,101 @@
+use vortex::{
+    stats::{ArrayStatisticsCompute, Stat, StatsSet},
+    ArrayTrait, AsArray, IntoCanonical,
+};
+use vortex_error::VortexResult;
+
+use super::ByteBoolArray;
+
+impl ArrayStatisticsCompute for ByteBoolArray {
+    fn compute_statistics(&self, stat: Stat) -> VortexResult<StatsSet> {
+        if self.is_empty() {
+            return Ok(StatsSet::new());
+        }
+
+        let bools = self.as_array_ref().clone().into_canonical()?.into_bool()?;
+        bools.compute_statistics(stat)
```

Comment:
note for future this could be optimized to avoid copy but for now this is enough